R/dettl.R

```diff
@@ -64,6 +64,10 @@ DataImport <- R6::R6Class(
 
     get_transformed_data = function() {
       private$transformed_data
+    },
+
+    get_import_path = function() {
```

Comment:
Nothing wrong at all with this way but if I want a read-only attribute I usually go with putting `path` into the public members and then at the end of the `initialize` function doing

```r
lockBinding("path", self)
```

which prevents it being modified.  It's a matter of taste and I know getters come very naturally from a Java perspective